packages/react-navigation/test/navigation-context.test.tsx

```diff
@@ -1,163 +1,150 @@
-import { type Span } from '@bugsnag/core-performance'
-import { InMemoryDelivery, VALID_API_KEY, createTestClient } from '@bugsnag/js-performance-test-utilities'
-import type BugsnagPerformance from '@bugsnag/react-native-performance'
```

Comment:
Do we still need a `BugsnagPerformance` mock or can that be removed now?

Reply:
💥